src/vm/stackwalk.cpp

```diff
@@ -1741,9 +1741,12 @@ StackWalkAction StackFrameIterator::Filter(void)
                         // was a caller of an already executed exception handler based on the previous exception trackers.
                         // The handler funclet frames are already gone from the stack, so the exception trackers are the
                         // only source of evidence about it.
+                        // The filter funclet is an exception though, its frame is always present on the stack when its parent
```

Comment:
> The filter funclet is an exception though

Can you use a different term instead of 'exception' here? Perhaps "special case"? 😄